olp-cpp-sdk-core/src/client/OlpClient.cpp

```diff
@@ -240,31 +240,44 @@ std::shared_ptr<http::NetworkRequest> OlpClient::CreateRequest(
 }
 
 NetworkAsyncCallback GetRetryCallback(
-    int current_try, int current_backdown_period, const RetrySettings& settings,
+    int current_try, int current_backdown_period,
+    long long accumulated_wait_time, const RetrySettings& settings,
```

Comment:
```suggestion
    std::chrono::milliseconds accumulated_wait_time, const RetrySettings& settings,
```
Prefer abstract types over built-in types.